letsencrypt-apache/letsencrypt_apache/configurator.py

```diff
@@ -898,40 +913,66 @@ def _enable_redirect(self, ssl_vhost, unused_options):
             logger.info("Redirecting vhost in %s to ssl vhost in %s",
                         general_vh.filep, ssl_vhost.filep)
 
-    def _verify_no_redirects(self, vhost):
-        """Checks to see if existing redirect is in place.
+    def _verify_no_letsencrypt_redirect(self, vhost):
+        """Checks to see if a redirect was already installed by letsencrypt.
 
-        Checks to see if virtualhost already contains a rewrite or redirect
-        returns boolean, integer
+        Checks to see if virtualhost already contains a rewrite rule that is
+        identical to Letsencrypt's redirection rewrite rule.
 
         :param vhost: vhost to check
         :type vhost: :class:`~letsencrypt_apache.obj.VirtualHost`
 
         :raises errors.PluginEnhancementAlreadyPresent: When the exact
                 letsencrypt redirection WriteRule exists in virtual host.
+        """
+        rewrite_path = self.parser.find_dir(
+                "RewriteRule", None, start=vhost.path)
+
+        dir_dict = {}
+        pat = r'.*(directive\[\d+\]).*'
+        for match in rewrite_path:
+            m = re.match(pat, match)
+            if m:
+                dir_id = m.group(1)
+                if dir_id in dir_dict:
+                    dir_dict[dir_id].append(match)
+                else:
+                    dir_dict[dir_id] = [match]
+
+        if dir_dict:
+            for dir_id in dir_dict:
+                if [self.aug.get(x) for x in dir_dict[dir_id]] in [
+                        constants.REWRITE_HTTPS_ARGS,
```

Comment:
This list of two long constants is very awkward and hard to read; maybe make it a concise name?